unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/TransactionHUD/TransactionListHUDView.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using DCL.TransactionHUDModel;
+using UnityEngine;
+
+public class TransactionListHUDView : MonoBehaviour
+{
+    [SerializeField]
+    private RectTransform transactionPanel;
+
+    public event Action<ITransactionHUD> OnTransactionAcceptedEvent;
+    public event Action<ITransactionHUD> OnTransactionRejectedEvent;
+
+    private const string VIEW_PATH = "TransactionListHUD";
+    private const string VIEW_CHILD_PATH = "TransactionHUD";
+    private const string VIEW_OBJECT_NAME = "_TransactionListHUD";
+
+    internal static TransactionListHUDView Create()
+    {
+        TransactionListHUDView view = Instantiate(Resources.Load<GameObject>(VIEW_PATH)).GetComponent<TransactionListHUDView>();
+        view.Initialize();
+        return view;
+    }
+
+    private void Initialize() { gameObject.name = VIEW_OBJECT_NAME; }
+
+    public void ShowTransaction(ITransactionHUD transaction, Model model = null)
+    {
+        if (transaction == null)
```

Comment:
ITransactionHUD can be null by design? if not this should throw an Exception at least